### PR TITLE
Provide hints on how to fix CI complaints.

### PR DESCRIPTION
**Pull request by fruffy**
Provide some helpful hints on how to fix the linter complaints.

### CODE REVIEW

**Review by thomascalvert-xlnx — APPROVED** [review]
Useful change for new contributors 👍

File: cmake/Linters.cmake [hunk 1]
```diff
@@ -71,7 +71,7 @@ if(NOT ${CLANG_FORMAT_CMD})
     set(CLANG_FORMAT_CMD clang-format)
     add_custom_target(
       clang-format
-      COMMAND xargs -a ${CLANG_FORMAT_TXT_FILE} -r -d '\;' ${CLANG_FORMAT_CMD} --verbose --Werror --dry-run -i --
+      COMMAND xargs -a ${CLANG_FORMAT_TXT_FILE} -r -d '\;' ${CLANG_FORMAT_CMD} --verbose --Werror --dry-run -i -- || echo ${RED}clang-format failed. Run \"make clang-format-fix-errors\" to fix the complaints.${COLOURRESET}
```
Comment (vlstill) [inline cmake/Linters.cmake:74]: Hmm, wouldn't this hide the return code of check? Therefore the test would succeed even if a problem was found?

**Review by vlstill — CHANGES_REQUESTED** [review]

File: cmake/Linters.cmake (hunk 1 above)
Comment (fruffy) [reply cmake/Linters.cmake]: Thanks for catching this. Getting sloppy recently...

**Review by vlstill — APPROVED** [review]

**Comment by vlstill** [comment]
(I've clicked the rebase button to unblock merging)

**Pull request merged**